fizzbuzz/C/IsabelleFizzBuzz/solution.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+int main() {
+   
+
+   int n;
+  scanf("%d", &n);
+
+
+for(int i = 1; i < (n+1); i++){
+if (i%15 == 0){
+printf("FizzBuzz\n");
+}
+	else  if (i%3 == 0){
+   	printf("Fizz\n");
+   }
+   	else if(i%5 == 0){
+   		printf("Buzz\n");
+   	}
+else{
+	printf("%d\n", i);
+}
+}
+ 
+   return 0;
+
+}
```

Comment:
small nit - try to cleanup this whitespace so this reads easier and these blocks are indented properly